static-assets/css/style.css

```diff
@@ -125,6 +119,20 @@ VIDEO BACKGROUND CSS
   text-align: center;
 }
 
+#home .banner {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+#home .banner .container {
+  position: absolute;
```

Comment:
does the RTE toolbar and everything render right?

Reply:
Those are basic text fields (not RTEs), other than that the fields are rendering properly